doc-content/jbpm-docs/src/main/asciidoc/BAM/DashbuilderRuntime-section.adoc

```diff
@@ -16,29 +16,145 @@ After login, users may upload dashboard files exported from {CENTRAL}
 . In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Dashbuilder Data Transfer*.
-. If you want to export all dashbuilder data as a ZIP file, click *Export all* on the *Dashbuilder Data Transfer* page.
-. If you want to select the data to be exported, click *Custom Export* on the *Dashbuilder Data Transfer* page.
+. On the *Dashbuilder Data Transfer* page, click *Custom export*.
+. In *Export Wizard* panel, verify the pages that you want to include in the ZIP file and click *Next*.
+. Select *Export Wizard* and click *Open*.
++
+You can see the dashbuilder runtime home page. If you are not logged in, you are redirected to the log in page.
+. Log in to Dashbuilder Runtime.
+. Go to *Dashboards → Runtime Dashboards* and select the exported ZIP file.
++
+The selected data is exported and dashbuilder runtime updates the model content when it is opened.
 
-image::BAM/GradualExport.png[]
 
 
 == Execution modes
 Dashbuilder Runtime has execution modes, which are:
 
-* *SINGLE*: This is the default mode. When a dashboard is uploaded then it will be used in Runtime. It is also possible to refer on dashboards under the directory configured using the system property `dashbuilder.import.base.dir`, default value is `/tmp/dashbuilder/models`. 
+* *SINGLE*: This is the default mode. When a dashboard is uploaded then it will be used in Runtime. It is also possible to refer on dashboards under the directory configured using the system property `dashbuilder.import.base.dir`, default value is `/tmp/dashbuilder/models`.
 If a file is in the `models` directory, then it is possible to load it using `RUNTIME_URL?import=FILE_NAME` in the URL. For example, the file `sales_dashboard.zip`, placed in the `models` directory, can be loaded using `RUNTIME_URL?import=sales_dashboard`(notice that the file extension is not necessary).
-* *STATIC*: In this mode Dashbuilder Runtime will not prompt users for upload and use the model file configured using system property `dashbuilder.runtime.import`. 
+* *STATIC*: In this mode Dashbuilder Runtime will not prompt users for upload and use the model file configured using system property `dashbuilder.runtime.import`.
 * *MULTI*:  Finally in multi mode users can upload multiple dashboards in the same Dashbuilder Runtime installation. To enable it the system property `dashbuilder.runtime.multi` must be set as true.
 
 image::BAM/DashbuilderRuntimeMultiMode.png[]
 
 == Embedding dashboards
 
-Dashboards running in Dashbuilder Runtime can be embedded in third-party applications using an `iframe`. To embed dashboards, the query parameter *standalone* should be added to the URL along with the dashboard page name using the query param *perspective*: 
+Dashboards running in Dashbuilder Runtime can be embedded in third-party applications using an `iframe`. To embed dashboards, the query parameter *standalone* should be added to the URL along with the dashboard page name using the query param *perspective*:
```

Comment:
In case of multi mode, you also need to specify which dashboard you want to use, using the property import={dashboard_name}, hence the resulting URL whould be RUNTIME_URL?standalone&import=DASHBOARD_NAME&perspective=PAGE_NAME